scripts/automate_release.py

```diff
@@ -0,0 +1,320 @@
+import json
+import os
+
+
+versions = os.environ['VERSIONS']
+linux_matricies = os.environ['LINUX_MATRICIES']
+macos_matricies = os.environ['MACOS_MATRICIES']
+windows_matricies = os.environ['WINDOWS_MATRICIES']
+
+#linux_matricies = os.getenv("LINUX_MATRICIES")
+#macos_matricies = os.getenv("MACOS_MATRICIES")
+#windows_matricies = os.getenv("WINDOWS_MATRICIES")
+
+def generate_json(commands):
+    matrix = '''
```

Comment:
I think better design would be to create json template file and then fill it with commands. or create a iteration that will generate these JSON files.